logdna/logdna.py

```diff
@@ -1,30 +1,33 @@
-import sys
-import time
 import json
 import logging
+import requests
 import socket
+import sys
 import threading
-import requests
+import time
+
 from requests.adapters import HTTPAdapter
 from urllib3.util import Retry
+
 from .configs import defaults
 from .utils import sanitize_meta, get_ip
 
-logger = logging.getLogger(__name__)
-
 class LogDNAHandler(logging.Handler):
     def __init__(self, key, options={}):
         logging.Handler.__init__(self)
 
+        self.internalHandler = logging.StreamHandler(sys.stdout)
```

Comment:
Sorry, didn't notice before but let's be Pythony and format this `internal_handler`